### PR TITLE
QDB-13081 - [cryptofeed] add support to more callbacks

### PR DESCRIPTION
**Pull request by igorniebylski**
changed plugin name from quasar to quasardb

added more channels to quasardb backend

added quasardb backend to setup.py,  INSTALL.md,  README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -141,6 +141,7 @@ Supported Backends:
 * Kafka
 * RabbitMQ
 * PostgreSQL
+* [QuasarDb](https://quasar.ai/)
```
Comment (solatis) [inline README.md]: QuasarDB

File: cryptofeed/backends/quasardb.py [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+
+    def _create_query(self):
+        self.query = f'CREATE TABLE "{self.table}" (exchange SYMBOL(exchange), symbol SYMBOL(symbol), id STRING, client_order_id STRING, side SYMBOL(side), status STRING, type STRING, price DOUBLE, amount DOUBLE, remaining DOUBLE, account STRING, receipt_timestamp TIMESTAMP) SHARD_SIZE = {self.shard_size}'
+
+
+class TransactionsQuasar(QuasarCallback, BackendCallback):
+    table_prefix = "transactions"
+
+    def _create_query(self):
+        self.query = f'CREATE TABLE "{self.table}" (exchange SYMBOL(exchange), currency SYMBOL(currency), type STRING, status STRING, amount DOUBLE, receipt_timestamp TIMESTAMP) SHARD_SIZE = {self.shard_size}'
```
Comment (solatis) [inline cryptofeed/backends/quasardb.py]: Couldn't / shouldn't `type` and `status` also be SYMBOL columns?

Same for teh other tables

File: cryptofeed/backends/quasardb.py [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+
+    def _create_table(self, conn):
+        if not conn.table(self.table).exists():
+            conn.query(self.query)
+
+    async def write(self, data: dict):
+        self._set_table_name(data)
+        self._create_query()
+        data = self.format(data)
+        df = qdbpd.DataFrame(data, columns=self.columns, index=[data['timestamp']])
```
Comment (solatis) [inline cryptofeed/backends/quasardb.py]: Does this even work? I did not know we re-exported `DataFrame` from quasardb.pandas

On that note, perhaps pulling in all of Pandas is a bit too heavy? Perhaps we can also do it with just numpy and use `quasardb.numpy` instead? 

As we're intending to merge this upstream, I think this is important.

**Comment by igorniebylski** [comment]
fixed naming, `write` uses qdbnp now

**Review by solatis — APPROVED** [review]

**Pull request merged**